web/js/containers/map-interactions/map-interactions.js

```diff
@@ -30,13 +30,15 @@ class MapInteractions extends PureComponent {
     const mapClasses = this.getMapClasses();
     return (
       <>
-        <ContextMenuTrigger id="context-menu-trigger">
-          <div id="wv-map" className={mapClasses} />
-        </ContextMenuTrigger>
         <RightClickMenu />
         <OlZoomButtons />
         <OlRotationButtons />
-        <OlCoordinates show={!isDistractionFreeModeActive} />
+        <ContextMenuTrigger id="context-menu-trigger">
+          <div id="wv-map" className={mapClasses} />
+        </ContextMenuTrigger>
+        {!isDistractionFreeModeActive && (
```

Comment:
I think there may have been a bad conflict resolution here, it looks like you undid my changes that should read as follows: 
```
<OlCoordinates show={!isDistractionFreeModeActive} />
```

Reply:
Fixing this should fix the broken e2e tests as well.  You can confirm locally by doing:

1. Go to info menu
2. Enter distraction free mode
3. Click the eyeball icon top right to exit distraction free mode

Presently, this results in an error on this branch.  Should be fixed with the above change addressed.

Reply:
This worked.  Good eye catching this.  I was manually accepting a lot of rebase conflicts and they were all in favor of my local changes vs remote changes.  I ended up accepting all the local changes blindly...and this is what I get lol.